meta/saiserialize.cpp

```diff
@@ -1,7 +1,11 @@
 #include "sai_serialize.h"
 #include "meta/sai_meta.h"
 #include "swss/tokenize.h"
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wshadow"
 #include "swss/json.hpp"
```

Comment:
Add pragma into header file directly so others could also benefit?

Reply:
this is in swss-common repo, also new update to json would need to keep this but sure i can add

Reply:
any follow up?

Reply:
working on it

Reply:
https://github.com/Azure/sonic-swss-common/pull/292